src/main.js

```diff
@@ -817,7 +839,12 @@ function initializeAfterAppReady() {
       return;
     }
 
-    // get the requesting webContents url
+    // is the request coming from trusted popup window
```

Comment:
are there any permissions needed not covered by `supportedPermissionTypes`?

Reply:
Media covers microphone and camera, and looks 'display-capture' is missing for screen sharing

Reply:
Media covers microphone and camera, I will handle with Screen Share in another PR, because it looks like it requires more work than just enable permission.